preprocessors/emotion-recognition/README.md

```diff
@@ -1,6 +1,6 @@
 # Expression Recognition Preprocessor
 
-This preprocessor has been created in order to detect expressions of an individual person. The output generated by a preprocessor would be ```happy, neutral or sad.``` 
+This preprocessor has been created in order to detect expressions of an individual person. The output generated by a preprocessor would be ```happy, neutral or sad.``` The code is built on [Deepface](https://github.com/serengil/deepface). There are several available models such as FaceNet, Openface, DeepFace etc. The code can be modified to use either of the models. The modifications need to be performed on `line 105` to specify which model should be used for analysis. 
```

Comment:
suggest "any of the models" if you mean any of the supported ones. Either is for choosing between just two options.

Pointing to a specific line number is fragile. Perhaps give the variable name the developer can search to find this setting? Or can this be set at the top of the file, as a configuration option?